Caltech-IPAC/firefly
# DM-7779: add unit test for FitsRead class

To test:
          1. check out this branch
          2. check out the data in firefly_test_data with the same branch name
          3. build and then run the unit test by :
                 gradle :firefly:test

Thanks!
